.github/workflows/post-pull-request-checks-automation.yml

```diff
@@ -40,26 +40,49 @@ on:
         type: number
         default: 1
         description: Configuration for the timeout-minutes parameter of the workflow.
+      run-only-latest-workflow:
+        type: string
+        description: 'Run only if the current workflow is the most recent workflow.'
+        default: 'true'
 
 jobs:
   post-pull-request-checks-automation:
+    outputs:
+      latest-run-number: ${{ steps.get-latest-workflow-run-number.outputs.latest_run_number }}
     runs-on: ubuntu-22.04
     timeout-minutes: ${{ inputs.timeout-minutes }}
     steps:
+      - name: Checkout
+        uses: Lombiq/GitHub-Actions/.github/actions/checkout@dev
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This shouldn't be needed.

Reply:
Without this it doesn't let me interact with GH CLI and GH run list doesn't work.

Reply:
What issue are you seeing, specifically?

Reply:
![image](https://github.com/Lombiq/GitHub-Actions/assets/92299130/8c74e81e-a661-4ba5-a885-231d68200e11)

Possible that now we use pwsh this might work. It was not working with a bash script that is for sure.
But even now with pwsh it is still alerting that the repo is messy.

Reply:
This is the same as: https://github.com/Lombiq/GitHub-Actions/pull/247#discussion_r1278604849

Reply:
The above line "failed to determine base repo: failed to run git: fatal: not a git repository"
WIll probably occure due to the `gh run list` command without a checkout.